### PR TITLE
EUI-4679 Global search date of death validation unhappy path

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4679

### Change description ###
Global Search - Date of death validation unhappy path

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by adnanakgun — APPROVED** [review]

File: src/search/containers/search-form/search-form.component.spec.ts [hunk 1]
```diff
@@ -2,7 +2,9 @@ import { HttpClientModule } from '@angular/common/http';
 import { NO_ERRORS_SCHEMA } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { FormBuilder } from '@angular/forms';
+import { RouterTestingModule } from '@angular/router/testing';
 import { Observable } from 'rxjs';
+import { SearchFormControl } from 'src/search/enums';
```
Comment (Daniel-Lam) [inline src/search/containers/search-form/search-form.component.spec.ts:7]: @johnbenjamin-hmcts - please change this to a relative import.

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - just one, very minor change that is needed.

**Comment by johnbenjamin-hmcts** [comment]
> @johnbenjamin-hmcts - just one, very minor change that is needed.

Thank you @Daniel-Lam, I have now applied the fix.

**Review by Daniel-Lam — APPROVED** [review]

**Pull request merged**